### ISSUE TITLE
Icon Request: TuneIn

### ISSUE THREAD

**Issue by qcountryknox** [FortAwesome/Font-Awesome#3159]
would like a tunein icon

**Comment by G-Driver** [comment]
+1

<img src="http://api.ning.com/files/8pnqwPqNt2MXyhc22D1dahRldNGHwTDpI7ziH*MKX1*3BDpE4QGh79uFPH00hfmedhR*EJ-6v-Ht2Q8XYBxenNmkjDe3gPBd/TuneIn_logo_white_1024.png">

**Comment by iSWORD** [comment]
+1

**Comment by subzerokangaroo** [comment]
+1

**Comment by CouponCodeSwap** [comment]
Yes, let's get this added. Is there something holding this up? If I can help, what needs to be done?

**Comment by q1068country** [comment]
wish we had a TuneIn Icon

**Comment by cliffjohnson** [comment]
Definitely a +1 for Tune In partners

**Comment by kricore** [comment]
Definetely a +1 as well

**Comment by stuardjay** [comment]
+1

**Comment by bitterac** [comment]
+1 please

**Comment by DmitryRomanov** [comment]
+1

**Comment by roirobo** [comment]
+1

Do you know why it hasn't been added? Since the issue is from 2014...

**Comment by Darrelk** [comment]
+1
Any updates on this?

**Comment by adiasandre96** [comment]
+1

**Comment by gavin2812** [comment]
+1

Hi, me too please guys! TuneIn is quite big now, please include a TuneIn icon for next release! Many thanks in advance!

**Comment by gavin2812** [comment]
+1

**Comment by GryfonN** [comment]
+1

**Comment by Perrin-S** [comment]
+1

**Comment by MikeWriting** [comment]
Bump please 
4 years?😳

**Comment by sergio97** [comment]
+1 Would love this icon.

**Comment by gjtyler** [comment]
+1 I need this for my project. I want to provide links to multiple platforms for my list of streamers.

**Comment by marianoarga** [comment]
5 years and counting

**Comment by webhelpus** [comment]
+1 and other blogger platforms

**Comment by leonardpg** [comment]
+1 the thought just crossed my mind to add a TuneIn icon to my footer. Looks like folks have been asking for years now.

**Comment by 14sprouj** [comment]
Need this please

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
